.github/workflows/JOB_tests.yml

```
@@ -12,6 +12,16 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.8", "3.9", "3.10", "3.11"]
+        exclude:
+          - os: macos-latest
+            version: "3.8"
+          - os: macos-latest
+            version: "3.9"
+        include:
+          - os: macos-13
+            version: "3.8"
+          - os: macos-13
+            version: "3.9"
```

This could solve this issue - https://github.com/v7labs/darwin-py/actions/runs/8830086377

```suggestion
        exclude:
          - os: macos-latest
            python-version: "3.8"
          - os: macos-latest
            python-version: "3.9"
        include:
          - os: macos-13
            python-version: "3.8"
          - os: macos-13
            python-version: "3.9"
```